.github/workflows/CI.yaml

```diff
@@ -13,18 +13,18 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout repository
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
 
       - name: Setup Flutter
-        uses: subosito/flutter-action@v1
+        uses: subosito/flutter-action@v2
         with:
           channel: stable
 
       - name: Install dependencies
         run: flutter pub get
 
       - name: Check formatting
-        run: flutter format . --output none --set-exit-if-changed
+        run: dart format . --output none --set-exit-if-changed
```

Comment:
`flutter format` has been deprecated for a while. `dart format` is its replacement.